### PR TITLE
Create aniruddh.json

### PR DESCRIPTION
**Pull request by aniruddhnishad**

### CODE REVIEW

File: sub-logs/aniruddh.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  "owner": {
+    "username": "aniruddhnishad",
+    "email": "mr.aniruddhnishad@gmail.com"
+  },
+
+  "target": {
+    "A": {
+       "name": "aniruddh", 
+       "value": "23.94.14.190"
```
Comment (WilliamDavidHarrison) [inline sub-logs/aniruddh.json:12]: 404?

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

File: sub-logs/aniruddh.json (hunk 1 above)
Comment (wclarkey) [reply sub-logs/aniruddh.json:12]: @WilliamDavidHarrison That is not descriptive enough to help anyone...

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]
We cannot reach your website.

**Review by WilliamDavidHarrison — APPROVED** [review]

**Review by wclarkey — APPROVED** [review]

**Comment by WilliamDavidHarrison** [comment]
Sorry for the confusion, I'm a new helper and didn't know what to do exactly.

**Pull request merged**